src/Inventory/InventoryList.js

```diff
@@ -105,27 +125,34 @@ class InventoryList extends Component {
 		this.setState({ newItemInputVisible: false, name: '' });
 	}
 
-	renderItem(item) {
-		return (
-			<ListItem
-				key={item._id}
-				title={item.name}
-				hideChevron
-			/>
-		);
-	}
-
 	renderItems() {
 		
 		let list = this.props.screenProps.inventoryLists.find(list => list._id === this.props.navigation.state.params.id);
 		let inventories = this.props.screenProps.inventories.filter(inventory => list.items.includes(inventory._id));
 
+		//TODO: convert SwipeListView to use the new FlatList version
+
 		if (inventories.length > 0) {
 			return (
 				<ScrollView style={{ flex: 1 }}>
-					<List>
-						{inventories.map(item => this.renderItem(item))}
-					</List>
+					<SwipeListView
+						dataSource={this.dataSource.cloneWithRows(inventories)}
+						renderRow={ data => (
+							<View style={styles.standaloneRowFront}>
+								<Text>{data.name}</Text>
+							</View>
+						)}
+						renderHiddenRow={ (data, secId, rowId, rowMap) => (
+							<View style={styles.standaloneRowBack} >
+								<Text style={styles.backTextWhite}></Text>
+								<TouchableOpacity onPress={() => this.deleteItem(data, rowMap, `${secId}${rowId}`)} >
+									<Text style={styles.backTextWhite}>Delete</Text>
+								</TouchableOpacity>
+							</View>
+						)}
+						disableRightSwipe
+						rightOpenValue={-75}
```

Comment:
What is this?

Reply:
The number of points from the right edge of the screen the hidden cell will be open by